src/server/scripts/Northrend/AzjolNerub/ahnkahet/boss_elder_nadox.cpp

```diff
@@ -234,47 +303,31 @@ class npc_ahnkahar_nerubian : public CreatureScript
     {
         npc_ahnkahar_nerubianAI(Creature* c) : ScriptedAI(c) { }
 
-        uint32 uiSprintTimer;
-        void Reset() override
-        {
-            if (me->GetEntry() == NPC_AHNKAHAR_GUARDIAN_ENTRY)
-                me->CastSpell(me, SPELL_GUARDIAN_AURA, true);
-            else // Swarmers
-                me->CastSpell(me, SPELL_SWARMER_AURA, true);
-
-            if (me->GetEntry() == NPC_AHNKAHAR_SWARMER || me->GetEntry() == NPC_AHNKAHAR_GUARDIAN_ENTRY)
-                me->SetInCombatWithZone();
 
-            uiSprintTimer = 10000;
-        }
-
-        void JustDied(Unit* /*killer*/) override
+        void Reset()
         {
-            if (me->GetEntry() == NPC_AHNKAHAR_GUARDIAN_ENTRY)
-            {
-                if (InstanceScript* pInstance = me->GetInstanceScript())
-                    if (Creature* nadox = ObjectAccessor::GetCreature(*me, pInstance->GetGuidData(DATA_ELDER_NADOX)))
-                        nadox->AI()->DoAction(ACTION_GUARDIAN_DIED);
-
-                me->RemoveAllAuras();
-            }
+            DoCastSelf(me->GetEntry() == NPC_AHNKAHAR_GUARDIAN ? SPELL_GUARDIAN_AURA : SPELL_SWARMER_AURA, true);
+            uiSprintTimer = 10000;
         }
 
-        void UpdateAI(uint32 diff) override
+        void UpdateAI(uint32 diff)
```

Comment:
why removing `override` ?

Reply:
it could be due of merging, maybe missed this change as some of merges has conflicts